proto/eth/v1/beacon_state.proto

```diff
@@ -32,15 +32,15 @@ option php_namespace = "Ethereum\\Eth\\v1";
 message BeaconState {
   // Versioning [1001-2000]
   uint64 genesis_time = 1001;
-  bytes genesis_validators_root = 1002 [(ethereum.eth.ext.ssz_size) = "32", (ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/beacon-chain/state/custom-types.Byte32"];
```

Comment:
We don't need custom types in proto. I added them in the first place because I initially wanted to fix tests as we go, which proved to be too difficult.